gufe/tests/test_protocol.py

```diff
@@ -67,7 +67,7 @@ def _execute(ctx, *, simulations, **inputs):
         )
 
 
-class DummySpecificSettings(settings.ProtocolSettings):
+class DummySpecificSettings(settings.Settings):
```

Comment:
@dotsdl @ijpulidos this is the key change here.  Rather than the protocol settings being inside `settings.Settings` , the protocol settings **are a subclass** of `settings.Settings`

Reply:
I think I was using this approach before in the NEQ cycling protocol, but I now can't remember what were the caveats or limitations using this. I will double check how these changes affect the protocol as soon as possible.

Is the idea here that we will then need just to instantiate a `ProtocolSettings` object and this will have all the `Settings` fields as well? This sounds simpler if that's the case.

Reply:
@ijpulidos yes.  So before you were defining `NonEqCyclingSettings` and putting that inside a `settings.Settings` object as the `protocol_settings=` argument.

so before:
```python
class NonEqCyclingSettings(settings.ProtocolSettings):
    eq_steps: int
    ...

class NonEqProtocol:
    @classmethod
    def default_settings(cls):
        return settings.Settings(..., protocol_settings=NonEqCyclingSettings(eq_steps=2500))
```

and with this PR: (the custom settings class is inheriting from something else now) 
```python
class NonEqCyclingSettings(settings.Settings):
    eq_steps: int
    ...

class NonEqProtocol:
    @classmethod
    def default_settings(cls):
        return NonEqCyclingSettings(eq_steps=2500)
```

this should remove a level of indentation when you're modifying settings specific to the protocol.